apps/web/lang/de.json

```diff
@@ -622,6 +622,16 @@
     "vat": {
       "incl": "inkl.",
       "plus": "zzgl."
+    },
+    "upload": {
+      "orderPropertyTypeFile":"Order-Eigenschaft – Typ Datei",
+      "uploading": "hochladen",
+      "uploaded": "hochgeladen",
+      "orderProperty":"Eigentum bestellen",
```

Comment:
Not used and wrong I guess wrong translated too.

Reply:
removed extra keyword,can you help me with german translations?